Basic FE project template/ui-src/styles/components/why-beleasing/_why-beleasing.scss

```diff
@@ -0,0 +1,159 @@
+.why-beleasing {
+  margin: 100px 100px;
+
+  @media screen and (max-width: 500px) {
+    margin: 55px;
+  }
+
+  &__container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: start;
+
+    @media screen and (max-width: 1000px) {
+      display: flex;
+      flex-direction: column-reverse;
+      justify-content: center;
+      align-items: center;
+    }
+
+    h1 {
+      @media screen and (max-width: 530px) {
+        font-size: 36px;
+      }
+
+      @media screen and (max-width: 320px) {
+        font-size: 28px;
+      }
+    }
+  }
+
+  &__title {
+    max-width: 980px;
+    width: 100%;
+    margin-right: 10%;
+    margin-top: 30px;
+
+    @media screen and (max-width: 1000px) {
+      max-width: 500px;
+      margin: 25px 0;
+      text-align: center;
+    }
+
+    @media screen and (max-width: 530px) {
+      text-align: center;
+    }
+
+    @media screen and (max-width: 375px) {
+      text-align: center;
+    }
+  }
+
+  p {
+    @media screen and (max-width: 1238px) {
+      font-size: 14px;
+      line-height: 20px;
+      font-weight: 400;
+      letter-spacing: 0px;
+    }
+  }
+}
+
+.why-beleasing__description {
+  .description {
+    margin-top: 35px;
+    margin-bottom: 60px;
+  }
+}
+
+.advantages {
+  display: flex;
+  list-style-type: none;
+  align-items: start;
+  padding: 0px;
+
+  @media screen and (max-width: 1238px) {
+    display: flex;
+    flex-direction: column;
+  }
+
+  @media screen and (max-width: 1000px) {
+    align-items: center;
+  }
+
+  &__item {
+    margin: 0 12px 40px 12px;
+
+    @media screen and (max-width: 1238px) {
+      display: flex;
+      flex-direction: row;
+      flex-wrap: wrap;
+      justify-content: flex-start;
+      align-items: center;
+    }
+
+    @media screen and (max-width: 1000px) {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      margin-bottom: 20px;
+    }
+  }
+
+  &__img {
+    @media screen and (max-width: 530px) {
+      width: 350px;
+    }
+
+    @media screen and (max-width: 375px) {
+      width: 300px;
+    }
+  }
+
+  .home-icon1 {
+    width: 80px;
+    height: 80px;
+    border: 1px solid $color--border;
+    border-radius: 100% 100%;
+    text-align: center;
+    margin-bottom: 25px;
+
+    @media screen and (max-width: 1238px) {
+      margin: 0;
+    }
+  }
+
+  .icon {
+    margin-top: 20px;
+  }
+
+  .title-text {
+    margin-bottom: 10px;
+
+    @media screen and (max-width: 1238px) {
+      font-size: 16px;
+      line-height: 23px;
+      letter-spacing: 0px;
+    }
+  }
+}
+
+.advantages__item:hover,
+.advantages__item:active {
+  transition: all 0.3s;
+  animation-name: floating;
+  animation-duration: 1s;
+  animation-iteration-count: one;
+  animation-timing-function: ease-in;
+}
+
+@keyframes floating {
```

Comment:
we can do something cool here: We can create a folder called utils afer the general folder in styles. Inside the new util folder, we can create a _keyframes.scss file and copy the code there, we could reuse it in other components later if needed. Do not forget to import the newly created _keyframes.scss file inside the style.scss before the _why-beleaseing.scss file. (the order matters, because the keyframes are used inside the components, if we import them after the components, we will get errors).